frontend/src/p2/TimeFramePage.js

```diff
@@ -7,20 +7,27 @@ import { Layout } from '../components/layout'
 import { TimeFrameInfoForm } from './forms/TimeFrameInfoForm'
 import { TrackPageViews } from '../TrackPageViews'
 import { getDoneForms } from '../utils/queriesAndMutations'
+import { Container } from '../components/container'
 import { BackButton } from '../components/backbutton'
 import { Steps } from '../components/stepper'
 
 const submitAndNavigate = (client, data) => {
-  client.writeData({ data: { scamInfo: JSON.stringify(data) } })
+  client.writeData({ data: { timeFrame: JSON.stringify(data) } })
   navigate(getDoneForms(client) ? 'confirmation' : 'whathappened')
 }
 
 export const TimeFramePage = () => (
   <Layout>
-    <BackButton route="/p2">
-      <Trans>Landing page</Trans>
-    </BackButton>
-    <Steps activeStep={1} totalSteps={6} />
+    <Container
+      display="flex"
+      width="90%"
+      flexDirection="row"
+      marginBottom="20px"
+    >
+      <Steps activeStep={0} steps={[{}, {}, {}, {}, {}, {}]} />
+    </Container>
+
+    <BackButton route="/p2" />
```

Comment:
Is this section supposed to change? This gives
<img width="460" alt="image" src="https://user-images.githubusercontent.com/8228248/64199715-e7a05880-ce58-11e9-9e32-5569131e9ef6.png">